defaults/main.yml

```diff
@@ -99,6 +99,8 @@ mailserver_replication_user_home: "{{ mailserver_mailbox_home }}"
 # here
 mailserver_mailbox_home: /var/mail
 
+mailserver_have_dkim: False
```

Comment:
Default to true for backwards compatibility?

Reply:
Sure thing ;)

Reply:
Done.